docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md

```diff
@@ -0,0 +1,78 @@
+# Upgrade to 8.0.0
+
+## New dependencies
+
+Depending on the used OS, run the corresponding command.
+
+### FreeBSD
+
+```sh
+pkg install p5-Plack-Middleware-ReverseProxy
+```
+
+### Debian / Ubuntu
+
+```sh
+sudo apt-get install libplack-middleware-reverseproxy-perl
+```
+
+### Centos
+
+```sh
+sudo cpanm Plack::Middleware::ReverseProxy
+```
+
+
+## Upgrading init scripts
+
+Go to the share folder:
+```
+cd `perl -MFile::ShareDir=dist_dir -E 'say dist_dir("Zonemaster-Backend")'`
+```
+
+And then install the `zm-rpcapi` daemon (`zm_rpcapi` on FreeBSD).
+
+### FreeBSD
+
+```
+su -l
+install -v -m 755 ./zm_rpcapi-bsd /usr/local/etc/rc.d/zm_rpcapi
+```
+
+### Debian / Ubuntu / CentOS
+
+```
+sudo install -v -m 755 ./zm-rpcapi.lsb /etc/init.d/zm-rpcapi
+```
+
+
+## Upgrading the database
```

Comment:
Can you add that depending of the database size, this upgrade can take some time (zonemaster.net (1.4 million test results) will take around 30min)?